### ISSUE TITLE
Create tokens using secret (string)

### ISSUE THREAD

**Issue by trevorhreed** [web-token/jwt-framework#121]
| Q                    | A
| -------------------- | -----
| Bug report?          | no
| Feature request?     | no
| BC Break report?     | no
| RFC? / Specification | no
| Version              | x.y(.z)

I'm trying to use this library to generate JWTs. In other languages/frameworks, I'm accustomed to passing in a secret (string). In this library, it appears I must construct a JWK. Since this tokens will be consumed in other projects, using other languages and frameworks, I'm trying to ensure I set things up correctly.

If I have a secret (say `abc123`) that I'm using in another project, how would I use that in this library? I noticed the documentation refers to [`JWKFactory::createFromSecret`](https://web-token.spomky-labs.com/component/jwk/jwk.html). But this seems to have been removed. Is there another way to accomplish what I'm trying to do?

**Comment by Spomky** [comment]
Hi,

The [`JWKFactory::createFromSecret`](https://github.com/web-token/jwt-framework/blob/v1.1/src/Component/KeyManagement/JWKFactory.php#L288-L299) method is available in the version `1.1` of the framework.
This version will be marked as stable by the end of the month (`v1.1.0@beta4` released few days ago).

Please note that this is just a convenient way to create a JWK (Json Web Key) from a secret. You can still do it like below:

```php
use Base64Url\Base64Url;
use Jose\Component\Core\JWK;

$jwk = JWK::create([
    'kty' => 'oct',
    'k'   => Base64Url::encode('abc123'),
    'alg' => 'HSxxx', // Optional but highly recommended
    'use' => 'sig', // Optional but highly recommended
    'kid' => 'Key #123', // Optional but highly recommended
]);
```

**Comment by trevorhreed** [comment]
That did it! Thank you!
